docs/en/guides/developer/debugging-memory-issues.md

```diff
@@ -0,0 +1,61 @@
+---
```

Comment:
I think it should be a KB article. wdyt?

Reply:
I think we should start having "official guides" vs. relying on KB articles.

In my opinion, KB articles should be predominantly used for Q&A-format articles.

Reply:
this section is for developer guides. this article is supposed to be used by all CH users.